src/components/Header.jsx

```diff
@@ -0,0 +1,7 @@
+export const Header = () => {
+  return (
+    <header>
+      <h1>Project Happy Thoughts 💭</h1>
+    </header>
+  );
+};
```

Comment:
You mentioned you wanted to use a reasonable amount of components, which I think you've managed very well. If you add the h1 in the App.jsx then you wouldn't need this component at all. 🤗